### ISSUE TITLE
Add timezone support

### ISSUE THREAD

**Issue by philipbl** [philipbl/Day-One-Lightroom-Plugin#16]

**Comment by thoughtgap** [comment]
Right now I have 1h difference between my LR-exported (with this tool) and directly imported photos. E.g. I have a photo in Lightroom taken on 17/02/2015 at 12:12 this plugin imports it to 13:12.
The photos are from 2 weeks ago and we have since changed to winter time (-1 hour), so there must be an issue with the current system time zone and the photo time zone I guess.

**Comment by bthelin** [comment]
I can add a work-around for this.  I noticed the issue too.  I live in Mountain Time and had a bunch of pictures taken in Pacific Time with my SLR.  I also added Apple Watch, iPhone, and written posts while in Pacific Time.  These posts were entered with the time zone attached and acted properly.  However, when I returned home, processed my images in Lightroom and posted them to Day One 2, the times were out of sync.

Before I go further, understand I'm not a programmer.  Just a huge fan.  After trying to figure out different people I could ask for help, I considered how either Lightroom or the plugin would be getting the time factor to apply to the entries.  The answer was the system time.  I altered my system's time zone to Pacific Time (where the images were shot) and exported again.  This time the correct time factor was applied and all the entries fell in line with the others.

**Long Story Short:** If you want a work-around, when you are ready to export the images from Lightroom, change your system's time zone to the time zone where the images were shot.  Export, then switch the time zone back.
